en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-400.md

```diff
@@ -4439,16 +4437,16 @@ Follow the instructions below to move all the existing API Manager configuration
             schema: "identity"
         ```
 
-    5.  Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_3.2.0_HOME>/repository/components/dropins` directory.
+    5.  Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_4.0.0_HOME>/repository/components/dropins` directory.
```

Comment:
need to skip following migrator configs also from the identity migration. 
    -
        name: "MigrationValidator"
        order: 2
    -
        name: "SchemaMigrator"
        order: 5
        parameters:
        location: "step2"
        schema: "identity"
    -
        name: "TenantPortalMigrator"
        order: 11  
        
        need to add these also in above step.